lib/util/httpc.ex

```diff
@@ -364,11 +366,18 @@ defmodule Antikythera.Httpc do
     :max_redirect, max ->
       {:max_redirect, max}
 
+    :connect_options, connect_options ->
+      {:connect_options, Enum.filter(connect_options, &filter_hackney_connect_option/1)}
+
     # :skip_body_decompression is used in processing response body, not here
     _, _ ->
       nil
   end
 
+  defunpt filter_hackney_connect_option(option :: :gen_tcp.connect_option()) :: v[boolean] do
+    option in [:inet, :inet6, :local]
```

Comment:
Do we need to supprt `:local`?

`:local` is [Unix Domain Socket](https://www.erlang.org/doc/man/gen_tcp.html#connect-4) . We need to pass `http+unix://` to Hackney to connect to a unix domain socket. However, `Httpc` only support `http(s)://` because `Httpc` uses `Url.t()` for URL and `Url.t()` [supports only the schemes](https://github.com/access-company/antikythera/blob/master/lib/type/string.ex#L106)

Reply:
Thank you.
For `Httpc`, only `:inet` and `:inet6` should be accepted.
Fixed at https://github.com/access-company/antikythera/pull/260/commits/8cd5f5154fadbbabc701b0a845d4149664cc6c07